Assets/GothicVR/Scripts/Creator/NpcCreator.cs

```diff
@@ -67,11 +68,10 @@ public static void Wld_InsertNpc(int npcInstance, string spawnpoint)
 
             if (PhoenixBridge.npcRoutines.TryGetValue(pxNpc.npcPtr, out List<RoutineData> routines))
```

Comment:
Can we get rid of this additional temp variable?

I mean we already have the data. We just need to look it up:
```csharp
newNpc.GetComponent<Routine>().routines.First();
```

Reply:
I don't get, what you are up to with this one